src/Microsoft.DotNet.Build.Tasks/PackageFiles/ilasm/ilasm.depproj

```diff
@@ -20,6 +20,7 @@
 
   <ItemGroup>
```

Comment:
You might consider renaming this project.

Reply:
I thought about that, but didn't come up with a good name. `AssemblyTools.depproj` perhaps? Any good suggestion?

Reply:
One thing though is that changing the project name might break some repos since the restore of this depproj happens on each individual repo, so we would have to react to the change on each one. Obviously, it's doable, but just think it's worth mentioning it.

Reply:
I was checking with @tannergooding 's PR that is changing to the sdk and looks like this project will soon go away anyways, so I'll keep the name for now just so that we don't have to change consuming repos twice. That said let me know if you still want me to do it so that I change it on a subsequent PR.